src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -209,7 +209,27 @@ private static void BindNonScalar(this IConfiguration configuration, object inst
         {
             if (instance != null)
             {
-                foreach (PropertyInfo property in GetAllProperties(instance.GetType()))
+                var modelProperties = GetAllProperties(instance.GetType()).ToList();
```

Comment:
var -> use explicit types

as per repo coding guideline:

> We only use var when it's obvious what the variable type is (e.g. var stream = new FileStream(...) not var stream = OpenStandardInput()).

in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

Reply:
I also prefer explicit types when it's not obvious - so it's great to hear that this repo prefers that too.